Automattic/wp-calypso
# Gutenberg Tracking: Track List View open and close (wpcom_block_editor_list_view_toggle)

#### Changes proposed in this Pull Request

* Track "List View" toggle in both Post/Page, and Site Editor. Event name: `wpcom_block_editor_list_view_toggle`. A property called `is_open` is also added to the event. The property indicates the new state of the list view.

#### Testing instructions



* Follow steps to enable tracking debugging at PCYsg-nrf-p2
* Open Post Editor, click on the List View button in the header. Make sure `wpcom_block_editor_list_view_toggle` is fired
* Open Site Editor, click on the List View button in the header. Make sure `wpcom_block_editor_list_view_toggle` is fired
* Make sure `is_open` is set to `true` when you open the list view, and it's `false` when you close the list view.



Related to https://github.com/Automattic/wp-calypso/issues/53410
